rest/src/db/CatapultDb.js

```diff
@@ -430,7 +430,8 @@ class CatapultDb {
 			if (filters.height)
 				conditions.push({ 'meta.height': convertToLong(filters.height) });
 
-			conditions.push({ 'meta.aggregateId': { $exists: false } });
+			if (!filters.embedded)
+				conditions.push({ 'meta.aggregateId': { $exists: false } });
```

Comment:
the default behaviour is of exclusion, so this condition will always need to be defined, something like:

```
conditions.push({ 'meta.aggregateId': { $exists: true === filters.embedded } });
```

the filter is not defined as mandatory in the params, so undefined needs to be considered as well, the suggested should also work with undefined

to test in `transactions()`:
- that default query without providing embedded, returns only non-embedded results
- that query with embedded=true, returns only embedded
- that query with embedded=false, returns only non-embedded results

i think the previous tests should not be working... because they should have been done with default behaviour (since this filter didn't exist), and with the updated version, there is no default behaviour, but since transcationRoutes is forcing a default to embedded (always exists, even if not provided it will be undefined and it evaluates to false) they pass

if the current code was like `if (false === filters.embedded`), which would have been the more precise way to do it, the old tests would probably fail (because embedded will be undefined) - tests failing is a desired behaviour because it would prove the code is working as expected